foundry.toml

```diff
@@ -56,6 +56,7 @@ polygon = "${ETH_NODE_URI_POLYGON}"
 polygonzkevm = "${ETH_NODE_URI_POLYGONZKEVM}"
 rootstock = "${ETH_NODE_URI_ROOTSTOCK}"
 sei = "${ETH_NODE_URI_SEI}"
+sonic = "${ETH_NODE_URI_SONIC}"
```

Comment:
why no entry in the bottom for the Etherscan key?

Reply:
fixed